### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ariefkasim**
Updated :
- overview picture
- documentation, license, and legal section
- build instruction in installation
- screenshots on build instruction

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -13,15 +13,28 @@
...
-This command builds the docker image of the service which is specified in the docker-compose.yml file.
+
+* Trough VSCode:  
+<kbd>CTRL</kbd>+<kbd>&uarr; SHIFT</kbd>+<kbd>P</kbd> or <kbd>F1</kbd> to open VSCode's command pallette and type `git clone`:
+
+![VS Code Git Clone command](graphics/git.png)
+
+### Build docker image
+
+- Navigate into `src` and find the file named `Dockerfile.example`. The `Dockerfile.example` is an example Dockerfile that can be used to build the Docker images of the backend and the frontend. If you choose to use these, rename them both to `Dockerfile` before proceeding
```
Comment (HummelN7872) [inline docs/installation.md:33]: There is no backend in this example, please remove this hint here.

File: README.md [hunk 2]
```diff
@@ -55,12 +56,12 @@ Login to Industrial Edge Device and click on Icon of the Hello-World Application
 
 ## Documentation
  
-- Here is a link to the [docs](docs/) of this application example.
 - You can find further documentation and help in the following links
   - [Industrial Edge Hub](https://iehub.eu1.edge.siemens.cloud/#/documentation)
   - [Industrial Edge Forum](https://www.siemens.com/industrial-edge-forum)
```
Comment (HummelN7872) [inline README.md:61]: Could you please use the link to the new Mendix based Forum here: 
https://forum.mendix.com/link/space/industrial-edge

**Review by HummelN7872 — COMMENTED** [review]
Please check my comments

File: docs/installation.md (hunk 1 above)
Comment (ariefkasim) [reply docs/installation.md:33]: ah yes of course, I used this template for other repos I have updated as well, will updated accordingly one by one and push right away

**Review by HummelN7872 — APPROVED** [review]

**Pull request merged**